feature/session/src/main/res/layout/item_speaker_session.xml

```diff
@@ -39,47 +39,57 @@
             app:layout_constraintGuide_end="24dp"
             />
 
-        <TextView
-            android:id="@+id/speaker_session_name"
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/session"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginTop="8dp"
             android:background="?attr/selectableItemBackground"
-            android:lineSpacingExtra="8sp"
-            android:text="@{speechSession.title.getByLang(lang)}"
-            android:textAppearance="@style/TextAppearance.DroidKaigi.Subtitle2"
-            app:isVisible="@{speechSession.hasSpeaker}"
             app:layout_constraintEnd_toEndOf="@id/guideline_end"
             app:layout_constraintStart_toStartOf="@id/guideline_start"
             app:layout_constraintTop_toTopOf="parent"
-            tools:text="DroidKaigi App Architecture"
-            />
+            >
 
-        <ImageView
-            android:id="@+id/time_icon"
-            android:layout_width="16dp"
-            android:layout_height="16dp"
-            android:layout_marginTop="8dp"
-            android:src="@drawable/ic_access_time_black_24dp"
-            app:isVisible="@{speechSession.hasSpeaker}"
-            app:layout_constraintStart_toStartOf="@id/guideline_start"
-            app:layout_constraintTop_toBottomOf="@id/speaker_session_name"
-            app:tint="?android:attr/textColorPrimary"
-            />
+            <TextView
+                android:id="@+id/speaker_session_name"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:lineSpacingExtra="8sp"
+                android:text="@{speechSession.title.getByLang(lang)}"
+                android:textAppearance="@style/TextAppearance.DroidKaigi.Subtitle2"
+                app:isVisible="@{speechSession.hasSpeaker}"
```

Comment:
Could you please change to set `isVisible` of @id/session parent view ?

Reply:
Thanks, I fixed it.